jujugui/static/gui/src/app/views/utils.js

```diff
@@ -1550,7 +1550,7 @@ YUI.add('juju-view-utils', function(Y) {
 
     @method deploy
     @param {Object} env Reference to the app env.
-    @param {Object} jem Reference to jem.
+    @param {Object} controllerAPI Reference to the controller api.
```

Comment:
Well, in JS they are all references to something, correct? ;-) What do you think about `The controller connection as an instance of juju.ControllerAPI.` ?